### PR TITLE
tests: Fix "Class 'WPCOM_VIP_CLI_Command' not found" error

### PR DESCRIPTION
**Pull request by sjinks**
This PR fixes errors found in tests during CI.

Ref: https://github.com/Automattic/vip-go-mu-plugins/actions/runs/4358243887/jobs/7619353489#step:5:382
Ref: https://github.com/Automattic/vip-go-mu-plugins/actions/runs/4358243887/jobs/7619353489#step:5:369

### CODE REVIEW

File: search/includes/classes/commands/class-healthcommand.php [hunk 1]
```diff
@@ -3,9 +3,9 @@
 namespace Automattic\VIP\Search\Commands;
 
 use \WP_CLI;
-use \WP_CLI\Utils;
 use WP_Error;
 
+require_once __DIR__ . '/../../../../vip-helpers/vip-wp-cli.php';
```
Comment (rebeccahum) [inline search/includes/classes/commands/class-healthcommand.php]: How come we're requiring this file now?

File: search/includes/classes/commands/class-queuecommand.php [hunk 2]
```diff
@@ -3,10 +3,10 @@
 namespace Automattic\VIP\Search\Commands;
 
 use \WP_CLI;
-use \WP_CLI\Utils;
 
 use \Automattic\VIP\Search\Queue\Schema;
 
+require_once __DIR__ . '/../../../../vip-helpers/vip-wp-cli.php';
```
Comment (rebeccahum) [inline search/includes/classes/commands/class-queuecommand.php]: Same ^

File: search/includes/classes/commands/class-versioncommand.php [hunk 3]
```diff
@@ -5,6 +5,8 @@
 use \WP_CLI;
 use \ElasticPress\Indexable as Indexable;
 
+require_once __DIR__ . '/../../../../vip-helpers/vip-wp-cli.php';
```
Comment (rebeccahum) [inline search/includes/classes/commands/class-versioncommand.php]: Same ^^

**Review by rebeccahum — CHANGES_REQUESTED** [review]

File: search/includes/classes/commands/class-healthcommand.php (hunk 1 above)
Comment (sjinks) [reply search/includes/classes/commands/class-healthcommand.php]: Please check this: https://github.com/Automattic/vip-go-mu-plugins/actions/runs/4358243887/jobs/7619353489#step:5:382

```
Error: Class 'WPCOM_VIP_CLI_Command' not found in /home/runner/work/vip-go-mu-plugins/vip-go-mu-plugins/search/includes/classes/commands/class-healthcommand.php:16
Stack trace:
#0 /home/runner/work/vip-go-mu-plugins/vip-go-mu-plugins/search/includes/classes/class-search.php(354): require_once()
#1 /home/runner/work/vip-go-mu-plugins/vip-go-mu-plugins/search/includes/classes/class-search.php(209): Automattic\VIP\Search\Search->load_dependencies()
#2 /home/runner/work/vip-go-mu-plugins/vip-go-mu-plugins/tests/search/includes/classes/test-class-settingshealthjob.php(20): Automattic\VIP\Search\Search->init()
```

It looks like it is possible that in some situations the class is not loaded, and it is safer to `require` the helper file explicitly. I wish we were using an autoloader :-(

File: search/includes/classes/commands/class-queuecommand.php (hunk 2 above)
Comment (sjinks) [reply search/includes/classes/commands/class-queuecommand.php]: Same :-)

File: search/includes/classes/commands/class-versioncommand.php (hunk 3 above)
Comment (sjinks) [reply search/includes/classes/commands/class-versioncommand.php]: Same :-)

File: search/includes/classes/commands/class-healthcommand.php (hunk 1 above)
Comment (rebeccahum) [reply search/includes/classes/commands/class-healthcommand.php]: Can we just require it on the test front then?

Comment (sjinks) [reply search/includes/classes/commands/class-healthcommand.php]: Done.

**Review by rebeccahum — APPROVED** [review]

**Pull request merged**